scss/_variables.scss

```diff
@@ -612,6 +612,7 @@ $form-check-input-width:                  1.25em !default;
 $form-check-min-height:                   $font-size-base * $line-height-base !default;
 $form-check-padding-left:                 $form-check-input-width + .5em !default;
 $form-check-margin-bottom:                .125rem !default;
+$form-check-label-color:                  $gray-700 !default;
```

Comment:
This should be `null` to start I'd think, since we don't style the `<label>`s to be another color—this gets inherited from the `<body>`.

Reply:
Yep, you're right. `inherit` it is.